### ISSUE TITLE
rate 0.001000, accuracy 7.0%, cross entropy 2.611571 

### ISSUE THREAD

**Issue by piascikj** [imdone/tensorflow#3507]
```

----
*opened via [imdone.io](https://imdone.io) from a code comment on [0c6fd1703eb8f990c8b071471b0105339ccf821d](https://github.com/imdone/tensorflow/commit/0c6fd1703eb8f990c8b071471b0105339ccf821d) by Pete Warden petewarden@google.com*


----

https://github.com/imdone/tensorflow/blob/8cb6e535f6ea14b380aa86d425169adad682cb8c/tensorflow/docs_src/tutorials/audio_recognition.md#L41-L47
